contributed_definitions/nyaml/NXsts.yaml

```diff
@@ -421,23 +413,47 @@ NXsts(NXsensor_scan):
         # parameters how to change the location from measurement to measurement during the scan
         scan_control(NXcollection):
           exists: optional
-          roi(NXtransformations):
+          scan_range(NX_NUMBER):
+            unit: NX_LENGTH
             exists: optional
+            doc: |
+              The scan range (in STM experiment) is the area that the tip will scan. (e.g.
+              5.000000E-9 5.000000E-9)
+          scan_offset(NX_NUMBER):
+            unit: NX_LENGTH
+            exists: optional
+            doc: |
+              The scan offset (in STM experiment) is the position of the tip in the scan area.
+              (e.g. -2.354637E-7 1.267476E-)
+          scan_direction:
+            exists: optional
+            doc: |
+              The scan direction (in STM experiment) is the direction from which side of the
+              sample the tip starts scanning.
+            enumeration: [down, up, top, bottom]
+          scan_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            exists: optional
+            doc: |
+              The angle of scan with the bottom or top side (depends on the scan_direction
```

Comment:
This docstring needs to be specified. Which angle is meant here? Between which entities?

Reply:
The angle will be measured wrt scan direction mentioned in doc.